### ISSUE TITLE
how to solo a track using the Daisy Chain modular mixer modules?

### ISSUE THREAD

**Issue by dandante** [sumpygump/quantal-audio#3]
Is it possible? If not consider this a feature request. Maybe it could be done with a long press on the Mute button?

**Comment by sumpygump** [comment]
Yes, this is a great idea. I was wanting to include a solo button too, but I'm running out of room :) My goal was to make a small form factor modular mixer. But maybe a dual purpose mute/solo button might be the route to go. I'll dig in and try it out.

**Comment by sumpygump** [comment]
Added solo option in v 2.2.1 (158c09b0e74cbf31578033c58add5ccf62ed0918)! You can activate this by long pressing on the mute button or by pressing 's' on the keyboard while mouse is hovering over the module.
